watcher/src/attestation_evidence_collector.rs

```diff
@@ -738,4 +757,54 @@ mod tests {
             sleep(Duration::from_millis(100));
         }
     }
+
+    #[test]
+    fn consensus_node_block_signer_from_dcap_evidence() {
+        // For DCAP evidence the signer key is the custom_identity.
```

Comment:
The test up above uses a `TestNodeClient`, which doesn't exercise the `get_block_signature()` so did these isolating unit tests here. Even though this module updated to use `EvidenceKind` in many places, the behavior only matters for the `get_block_signature()`